cmd/main.go

```diff
@@ -22,22 +22,40 @@ import (
 	"sigs.k8s.io/kubebuilder/v3/pkg/cli"
 	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	commonv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/v2"
+	commonv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/common/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
+	golangv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/base/v2"
+	golangv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/base/v3"
 	declarativev1 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/declarative/v1"
-	pluginv2 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v2"
-	pluginv3 "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v3"
 )
 
 func main() {
+
+	// Bundle plugin which built the golang projects scaffold by Kubebuilder go/v2
+	gov2Bundle, _ := plugin.NewBundle(golang.DefaultGoNameQualifier, plugin.Version{Number: 2},
+		commonv2.Plugin{},
+		golangv2.Plugin{},
+	)
+
+	// Bundle plugin which built the golang projects scaffold by Kubebuilder go/v3
+	gov3Bundle, _ := plugin.NewBundle(golang.DefaultGoNameQualifier, plugin.Version{Number: 3},
+		commonv3.Plugin{},
```

Comment:
`common` is ambiguous. Why not `kustomize-config`?

Reply:
I'd like a more generic name to make clear that is the base for any plugin language. So, wdyt about base?

Reply:
I could be convinced of `base.kustomize-config`, much like `base.go` is now a thing and reasonable.

Reply:
Camila's point is that it may be more things than kustomize configs. Like a base Makefile, or a LICENSE file. `seed`? `root`? `foundation`? I'm just spamming the first thing that goes through my head.

Reply:
The current Makefile is intimately tied to both `kustomize` and `go`. Ideally, the `kustomize`-related and `go`-related portions would be split into separate Makefile fragments, then combined at runtime. In practice this is kinda difficult to do because of shared variables so I don't necessarily recommend it right now.

The license is only applied to Go files, so that would go in the `base.go` plugin.

These are specific cases, but help enforce my point that files not of the same type as those of the "meat" of the host plugin, denoted by name in `<name>.kubebuilder.io`, are still related to that type.

Reply:
I was not talking to the boilerplate license, but a LICENSE file. I feel like `base.kubebuilder.io` and `base.go.kubebuilder.io` are not too different. However, users will never have to use `base.go.kubebuilder.io` as it is not even being registered in the CLI, the bundle `go.kubebuilder.io` is the one being registered, so we could use `base.kubebuilder.io`.

Reply:
>I was not talking to the boilerplate license, but a LICENSE file

Ah ok. In that case, the LICENSE file could just as easily be in the go base plugin, or a separate "common" plugin that doesn't contain any kustomize (or other) configuration files. Contrapositively, say I do call the kustomize config plugin "common", then create a config-gen plugin to replace kustomize configs with a config-gen config; what should I call the config-gen plugin, which also needs a LICENSE file but can't be paired with the "common" plugin?

>users will never have to use base.go.kubebuilder.io as it is not even being registered in the CLI, the bundle go.kubebuilder.io is the one being registered, so we could use base.kubebuilder.io.

True, since the "common" plugin is allowed as a first-class plugin, wouldn't it make sense to be more specific about what it contains? How about `common.kustomize`?

Reply:
We have the idea of having a `config-gen` plugin. @estroz is working on that.
So, I change the name of it to `config-base` and then, I hope that makes everybody happy. 

However, if not. I am OK with any name for it which does not have `kustomize` on that. Why? It brings the idea that is == golang plugins but using the only kustomize when is not. The idea of this plugin is to scaffold a common-base, common-configuration valid no matter the language.

Reply:
Your argument against using `kustomize` in the plugin name is not convincing. You can re-use `kustomize` in any plugin name and add more information to the name, so there really are no restrictions here.

Reply:
Closing that since it was decided via the pool. Thank you folks,